services/movie.py

```diff
@@ -0,0 +1,40 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet | Movie:

```

Comment:
The return type annotation `QuerySet | Movie` is incorrect for the `get_movies` function. This function should only return a `QuerySet`, not a single `Movie` instance. Update the return type to `QuerySet`.